### PR TITLE
Added Custom Textformfield widget & Custom Button Widget

### PR DESCRIPTION
**Pull request by ReddyGona**

### CODE REVIEW

File: lib/src/core/styles/app_colors.dart [hunk 1]
```diff
@@ -10,4 +10,7 @@ class AppColors {
   static Color errorTextField = Colors.red;
   static Color button = const Color(0xff6a12ef);
   static Color black = Colors.black;
+  static Color lightGrey = const Color(0xFFD0D0D0);
+  static Color textHintColor = const Color(0xFF545863);
```
Comment (saigopin) [inline lib/src/core/styles/app_colors.dart:14]: remove  color word here

File: lib/src/core/utils/injections.dart [hunk 2]
```diff
@@ -3,7 +3,7 @@ import 'package:flutter_clean_architecture/src/core/network/dio/dio_operations.d
 import 'package:flutter_clean_architecture/src/features/home/articles_injections.dart';
 import 'package:flutter_clean_architecture/src/shared/app_injections.dart';
 import 'package:get_it/get_it.dart';
-import 'package:secure_shared_preferences/secure_shared_preferences.dart';
+import 'package:secure_shared_preferences/secure_shared_pref.dart';
```
Comment (saigopin) [inline lib/src/core/utils/injections.dart:6]: why this changed bro? any issue

File: lib/src/shared/presentation/widgets/custom_button_widget.dart [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+  @override
+  Widget build(BuildContext context) {
+    return Align(
+      alignment: alignment,
+      child: SizedBox(
+        width: width,
+        child: ElevatedButton(
+          onPressed: onPressed,
+          style: ButtonStyle(
```
Comment (saigopin) [inline lib/src/shared/presentation/widgets/custom_button_widget.dart:34]: please move this style to common place

File: lib/src/shared/presentation/widgets/text_form_field_widget.dart [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+  final int maxLines;
+  final bool obscureText;
+  final String obscuringCharacter;
+  final bool readOnly;
+  final String? Function(String?)? validator;
+
+  const TextFormFieldWidget({
+    Key? key,
+    required this.hintText,
+    this.errorMessage = 'Invalid Input',
```
Comment (saigopin) [inline lib/src/shared/presentation/widgets/text_form_field_widget.dart:29]: get string for AppStrings

File: lib/src/core/utils/injections.dart (hunk 2 above)
Comment (ReddyGona) [reply lib/src/core/utils/injections.dart:6]: getting error when latesh pull is taken, i will revert it

**Pull request merged**